pkg/threescale/proxy_conf_test.go

```diff
@@ -152,7 +152,7 @@ func TestProxyConfigCacheFlushing(t *testing.T) {
 
 	testStopNotStartedErr := c.conf.systemCache.StopFlushWorker()
 	if testStopNotStartedErr == nil {
-		t.Fatalf("expected to get error when stopping unstarted worker")
+		t.Fatalf("expected to Get error when stopping unstarted worker")
```

Comment:
this is the wrong `get`

Reply:
fixed! I thought the refactoring tool was smarter... Sorry!